src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
```

Comment:
It's a good practice to check if the element exists before using it. If the '.logo' element does not exist, the `querySelector` method will return null, and any subsequent method calls or property accesses will result in a TypeError.